src/Tools/BuildUtil/BuildUtil/BuildUtil.csproj

```diff
@@ -9,7 +9,7 @@
     <OutputType>Library</OutputType>
     <RootNamespace>Roslyn.MSBuild.Util</RootNamespace>
     <AssemblyName>Roslyn.MSBuild.Util</AssemblyName>
-    <TargetFramework>netstandard1.3</TargetFramework>
+    <TargetFramework>net46</TargetFramework>
```

Comment:
For now it's windows only. I think it will quickly get used in our Linux builds as well though. But I wanted to land the big multi-targeted changed first before I added multi-targeting here.